File: javascript-closure-angular-client/API/Client/LegalReasonReferencedProcesses.js
goog.provide('API.Client.LegalReason_referencedProcesses');

/**
 * @record
 */
API.Client.LegalReasonReferencedProcesses = function() {}

/**
 * Process Identifier (NFe <nProc> tag)
 * @type {!string}
 * @export
 */
API.Client.LegalReasonReferencedProcesses.prototype.nProc;

/**
 * Process Origin (NFe <indProc> tag) - '0' # SEFAZ; - '1' # Justiça Federal; - '2' # Justiça Estadual; - '3' # Secex/RFB; - '9' # Outros 
 * @type {!string}
 * @export
 */
API.Client.LegalReasonReferencedProcesses.prototype.indProc;

/** @enum {string} */
API.Client.LegalReasonReferencedProcesses.IndProcEnum = { 
  0: '0',
  1: '1',
  2: '2',
  3: '3',
  9: '9',
}

File: async-scala-client/src/main/scala/io/swagger/client/model/LegalReason_referencedProcesses.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class LegalReason_referencedProcesses (
  nProc: Option[String],  // Process Identifier (NFe &lt;nProc&gt; tag)
indProc: Option[String]  // Process Origin (NFe &lt;indProc&gt; tag) - &#39;0&#39; # SEFAZ; - &#39;1&#39; # Justiça Federal; - &#39;2&#39; # Justiça Estadual; - &#39;3&#39; # Secex/RFB; - &#39;9&#39; # Outros 
)
